### DISCUSSION TITLE
GoPro 12 external power supply

### DISCUSSION THREAD

**Discussion by christophefouquin** [gopro/labs#628] [General]
Hello everyone,
I power my GP8 and my GP10 with the battery from my DJI AVATA and with my new GP12 it doesn't work, it doesn't turn on.
I activated TUSB but nothing happened.
Thank you in advance for your help.
Christopher

**Comment by dnewman-gpsw** [comment]
What does it do?  What is the error message?  Does it do the same from a USB power brick?

**Reply by christophefouquin** [reply]
Thanks for your feedback. From a USB power supply it works with TUSB, but with the battery (DJI AVATA) nothing happens at all. No light on, no message, nothing!

**Reply by dnewman-gpsw** [reply]
You will need to check the point with meter, as it should work fine.

**Reply by christophefouquin** [reply]
on the multimeter I have a voltage of 14.8V

**Reply by dnewman-gpsw** [reply]
At the USB port, 14.8V would kill a camera as it uses 5V.  You might need something like one of these https://www.amazon.com/s?k=usb+voltage+tester
